src/v0/destinations/customerio/transform.js

```diff
@@ -113,6 +115,7 @@ function processSingleMessage(message, destination) {
       logger.error(`could not determine type ${messageType}`);
       throw new InstrumentationError(`could not determine type ${messageType}`);
   }
```

Comment:
this is redundant now as string type check for evName is already handled in validateEventName

Reply:
Added the validateEventName check only for track event but evName we are mapping from page name or page url as well in case of page events.